v2g-liberty-cards/src/charger-settings-card.ts

```diff
@@ -27,6 +27,8 @@ export class ChargerSettingsCard extends LitElement {
   @state() private _useReducedMaxPower: HassEntity;
   @state() private _chargerMaxChargingPower: HassEntity;
   @state() private _chargerMaxDischargingPower: HassEntity;
+  @state() private _quasarLoadbalancerLimit: HassEntity;
```

Comment:
Since "load" and "balancer" are separate words, I propose to capitalise "Balancer".

Reply:
OK

Reply:
OK